### ISSUE TITLE
Add a helper 

### ISSUE THREAD

**Issue by nicofrand** [jsilvestre/tasky#39]
This might be silly but I remember wondering how to add a tag to tasky the first time I used it.
Maybe a simple helper saying to use # would be nice ?

**Comment by jsilvestre** [comment]
You are absolutely right, some important features are not discoverable *at all*. Like adding a tag or sorting tasks.
